compiler-rt/lib/builtins/cpu_model/aarch64/fmv/windows.inc

```diff
@@ -0,0 +1,99 @@
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <processthreadsapi.h>
+#include <stdint.h>
+
+#ifndef PF_ARM_V82_DP_INSTRUCTIONS_AVAILABLE
+#define PF_ARM_V82_DP_INSTRUCTIONS_AVAILABLE -1
```

Comment:
Is there some reason not to just use the actual numbers from the SDK, instead of -1?  The new numbers should work even if you're using an old SDK.  And then we don't need to worry about accidentally reporting features are missing if someone builds with an old SDK.

Reply:
I wasn't really sure if it is a good idea or not. Thanks for convincing me.